README.md

```diff
@@ -18,6 +18,24 @@ IPFS API wrapper library in JavaScript
 $ npm install --save ipfs-api
 ```
 
+#### Running the daemon with the right port
+
+To interact with the API, you need to have a local daemon running (at the moment; we have plans to make it work independently). It needs to be open on the right port: `5001`. This is the default.
```

Comment:
> local daemon running (at the moment; we have plans to make it work independently

What do you have in mind?

> It needs to be open on the right port: `5001`. This is the default.

The default port is 5001, but it can open any really, if the user changes the default port, then it has to change the port in the example too :)